bee-message/src/output/alias.rs

```diff
@@ -217,13 +234,31 @@ impl AliasOutput {
     ///
     #[inline(always)]
     pub fn state_controller(&self) -> &Address {
-        &self.state_controller
+        if let UnlockCondition::StateControllerAddress(address) = self
+            .unlock_conditions
+            .get(StateControllerAddressUnlockCondition::KIND)
+            .unwrap()
```

Comment:
These unwraps are safe too because they are part of every `AliasOutput`?

Reply:
Yeah, I moved the comment up so that it's clearer it applies to the whole function

Reply:
test

Reply:
est2